Добавил домашенне задание 11.145 в. Условие: дан массив из n%2=0 эл-ов.

alexfedorov10 opened the pull request:
Поменять местами его половины след. об.: 1й эл. с последним, 2й с
предпоследним...

Commit:
Добавил домашенне задание 11.145 в. Условие: дан массив из n%2=0 эл-ов.
Поменять местами его половины след. об.: 1й эл. с последним, 2й с
предпоследним...

strazhkoe commented on src/Main.java:
```diff
@@ -0,0 +1,22 @@
+		System.out.println(Arrays.toString(mas));
+
+	   	}
+	}
```
Сделай пожалуйста Ctrl+Shift+F. Непонятно, к чему относится эта закрывающая скобка.

strazhkoe reviewed (changes requested):
Непонятно, что сделано в этом коде. Даже прочитав условие и запустив код, понятней не стало. Добавь, пожалуйста, комментарии в код и в вывод на экран пользователю.

---

Upd. увидел условие в заголовке пул реквеста. Теперь стало понятно

Commit:
Сделал Ctrl+Shift+F  в домашеннме задании №11.145 в) 
Условие: дан массив из n%2=0 эл-ов.
Поменять местами его половины след. об.: 1й эл. с последним, 2й с
предпоследним...

Commit:
Добавил решение задания 12.28-29. Программа заполняет квадратный массив
любой размерности n*n по спирали числами от 1 до n .

strazhkoe commented:
Не надо было самому сливать в мастер. Я еще не посмотрел результат последнего коммита
